tutorials/cap-extend-sfsf-add-logic/cap-extend-sfsf-add-logic.md

```diff
@@ -194,6 +196,8 @@ In the **Terminal** type `cd ..` and press **Enter** to go back to the **project
 
 ![Figure 9 – Change back to project root](change-to-project.png)
 
+>**NOTE**: newer versions of CDS have handed over to **SAP Cloud SDK** the creation of HTTP clients for making HTTP requests to external services, using the `@sap-cloud-sdk/http-client` node package. So, if you jump-started your CAP project, **before such CDS update**, that dependency might not have been included in your `package.json` file, and, thus, not installed when you ran `npm install`. Before testing the handler, please verify that you have `@sap-cloud-sdk/http-client` listed in the dependencies of your `package.json` and, if not, run: `npm install @sap-cloud-sdk/http-client`.
```

Comment:
Please don't start a note with the label "NOTE". The formatting already makes it a note. If you want to emphasize the note you can use other labels, for example, "CAUTION", "IMOPORTANT" etc.

Reply:
Done.